packages/types/index.d.ts

```diff
@@ -163,6 +163,14 @@ export interface IUser {
 // Subset of HTML autocomplete values.
 type AutoCompleteValue = 'off' | 'url' | 'email'
 
+export interface IFieldVisibilityCondition {
+  fieldKey: string
+  fieldValue: string
+
+  // This is synced with FieldVisibilityOp GraphQL enum.
+  op: 'equals' | 'not_equals' | 'always_true'
```

Comment:
nit: should this be a type i.e. `type FieldOp = 'equals' | 'not_equals' | 'always_true'`

Reply:
thanks!